### PR TITLE
Unify location for assets.. again..

### PR DESCRIPTION
**Pull request by Soitora**
Realized that keeping them seperate was a bad idea, especially for newcommers that want to contribute.

### CODE REVIEW

File: src/sandbox/style-guide.md [hunk 1]
```diff
@@ -190,14 +190,14 @@ Many of the stuff previously mentioned generally applies when making guides too,
...
-> 	<img height="400" intrinsicsize="208x500" width="260" :src="$withBase('/help/guides/reader-settings/assets/Reader-Background-Black.png')">
+> 	<img height="400" intrinsicsize="208x500" width="260" :src="$withBase('/assets/guides_reader-Background-Black.png')">
 > 	<h4>Black</h4>
 > 	<p>Surrounds page with black. With an AMOLED screen you can save battery by using this.</p>
 > </figure>
 > </CarouselItem>
 > <CarouselItem>
 > <figure class="centered">
-> 	<img height="400" intrinsicsize="208x500" width="260" :src="$withBase('/help/guides/reader-settings/assets/Reader-Background-White.png')">
+> 	<img height="400" intrinsicsize="208x500" width="260" :src="$withBase('/assets/guides_reader-Background-White.png')">
```
Comment (ghostbear) [inline src/sandbox/style-guide.md:200]:
```suggestion
> 	<img height="400" intrinsicsize="208x500" width="260" :src="$withBase('/assets/guides_reader-background-white.png')">
```
Didn't you rename it to this?

File: src/sandbox/style-guide.md [hunk 2]
```diff
@@ -190,14 +190,14 @@ Many of the stuff previously mentioned generally applies when making guides too,
 > <Carousel name="carousel-background-color">
 > <CarouselItem>
 > <figure class="centered">
-> 	<img height="400" intrinsicsize="208x500" width="260" :src="$withBase('/help/guides/reader-settings/assets/Reader-Background-Black.png')">
+> 	<img height="400" intrinsicsize="208x500" width="260" :src="$withBase('/assets/guides_reader-Background-Black.png')">
```
Comment (ghostbear) [inline src/sandbox/style-guide.md:193]:
```suggestion
> 	<img height="400" intrinsicsize="208x500" width="260" :src="$withBase('/assets/guides_reader-background-black.png')">
```
Didn't you rename it to this?

File: src/sandbox/style-guide.md (hunk 1 above)
Comment (Soitora) [reply src/sandbox/style-guide.md:200]: This is correct

File: src/sandbox/style-guide.md (hunk 2 above)
Comment (Soitora) [reply src/sandbox/style-guide.md:193]: This is correct

**Review by ghostbear — APPROVED** [review]

**Pull request merged**